src/index.html

```diff
@@ -19,7 +19,7 @@ <h1>What's Cook'n</h1>
         <h2 class='greeting'> Welcome, User </h2>
         <div class='search-area'>
           <img class='search-bar search-image' src='https://image.flaticon.com/icons/svg/660/660029.svg' alt='Search button'>
-          <input id='search-input' class='search-bar search-input searchbar' type='text' placeholder='Search recipes by name or ingredients' value=''>
+          <input aria-label='searchbar' id='search-input' class='search-bar search-input searchbar' type='text' placeholder='Search recipes by name or ingredients' value=''>
```

Comment:
Amazing that one label gave us such big movement on the Lighthouse score. Nice work.